### ISSUE TITLE
genericName wrongly interpreted for higher ranks e.g Foraminifera

### ISSUE THREAD

**Issue by gbif-portal** [gbif/portal-feedback#3961]
**genericName wrongly interpreted for higher ranks e.g Foraminifera**

-----

User: [See in registry](https://www.gbif.org/api/feedback/user/c0e33a58fae9e420fe2a843421926dba:68287d7d8eb8aa744cf87cdbfd1faddbd8655fff63caeff34558b3e89ad6557b978a5925501328b8920bfae57d19f1b43513d1cbb429d1de2142bc91bc41ca8a)
System: Safari 15.1.0 / Mac OS X 10.15.7
Referer: https://www.gbif.org/occurrence/1061190602
Window size: width 1466 - height 964
[API log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2022-03-03T15:56:56.365Z',to:'2022-03-03T16:02:56.365Z'))&_a=(columns:!(_source),filters:!(),index:'3390a910-fcda-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
[Site log](https://private-logs.gbif.org/app/discover#/?_g=(filters:!(),refreshInterval:(pause:!t,value:0),time:(from:'2022-03-03T15:56:56.365Z',to:'2022-03-03T16:02:56.365Z'))&_a=(columns:!(_source),filters:!(),index:'5c73f360-fce3-11ea-a9ab-4375f2a9d11c',interval:auto,query:(language:kuery,query:''),sort:!()))
System health at time of feedback: OPERATIONAL
datasetKey: 58d0f326-2e85-4d0a-a744-571461220f00
publishingOrgKey: 0dc9c7a0-b5c6-11d9-a0b8-b8a03c50a862
Network keys: 99d66b6c-9087-452f-a9d4-f15f2c2d0e7e 
Node handles: @albenson-usgs

**Comment by ManonGros** [comment]
I don't think this is a data content issue, I think this is an interpretation issue.
The generic name refers to the first part of a binomial name of the species and shouldn't be inferred for higher rank names like Foraminifera (which is a phylum).

<img width="826" alt="Screenshot 2022-04-26 at 15 03 27" src="https://user-images.githubusercontent.com/7677271/165306105-492c556b-dadf-4c7b-9ba3-b9ca69b7ebe3.png">

It should be left blank instead.
